packages/web/styles/globals.css

```diff
@@ -335,3 +335,15 @@
 .breakspaces {
   white-space-collapse: break-spaces;
 }
+
+.folder-left {
+  mask-image: url("./folder-left-tab.svg");
+  mask-size: contain;
+  mask-repeat: no-repeat;
+}
+
+.folder-right {
+  mask-image: url("./folder-right-tab.svg");
+  mask-size: contain;
+  mask-repeat: no-repeat;
+}
```

Comment:
IMO we should avoid putting feature-specific styles in globals.css. (Even though there may be some there already). I think this is more fit for defining our design system instead. In isolation it's not a big deal, but over the long term it could be come quite messy. Thoughts on finding a way to co-locate these styles with the markup?